customer-commons/src/test/java/no/unit/nva/customer/model/CustomerTest.java

```diff
@@ -29,6 +29,7 @@ public void customerMappedToJsonAndBack() throws JsonProcessingException {
                 .withInstitutionDns("institution.dns")
                 .withAdministrationId("adminstr@ion.id")
                 .withFeideOrganizationId("123456789")
+                .withCristinId("http://cristin.id")
```

Comment:
I would add the following to the test so that it fails if in future updates we forget to add a new field to the test

```
        assertThat(customer,doesNotHaveNullOrEmptyFields());
```